### PR TITLE
Add golang to k3d vm install

### PR DESCRIPTION
**Pull request by barchw**
**Description**
Add golang installation to k3d vm for executing go tests

**Related issue(s)**
https://github.com/kyma-project/test-infra/issues/5411

### CODE REVIEW

File: development/custom-image/install-deps-debian.sh [hunk 1]
```diff
@@ -102,6 +102,19 @@ sudo apt-cache madison google-fluentd
 sudo apt-get install -y 'google-fluentd=1.*'
 sudo apt-get install -y google-fluentd-catch-all-config
 
+# install go
+ export GO_VERSION=1.18.2
```
Comment (Halamix2) [inline development/custom-image/install-deps-debian.sh:106]: Can you put `GO_VERSION` at the top with the rest of dependencies versions?

File: development/custom-image/install-deps-debian.sh [hunk 2]
```diff
@@ -102,6 +102,19 @@ sudo apt-cache madison google-fluentd
...
+
+ export GOPATH=/workspace/go
+ export PATH=$GOPATH/bin:/usr/local/go/bin:$PATH
```
Comment (Halamix2) [inline development/custom-image/install-deps-debian.sh:109]: Image creation creates new VM based on Debian image, runs this script, stops the VM and creates new image from the VM. Exporting variables won't survive VM shutdown.

File: development/custom-image/install-deps-debian.sh [hunk 3]
```diff
@@ -102,6 +102,19 @@ sudo apt-cache madison google-fluentd
 sudo apt-get install -y 'google-fluentd=1.*'
 sudo apt-get install -y google-fluentd-catch-all-config
 
+# install go
+ export GO_VERSION=1.18.2
+
+ export GOPATH=/workspace/go
```
Comment (Halamix2) [inline development/custom-image/install-deps-debian.sh:108]: Does `/workspace` exist?

**Review by Halamix2 — CHANGES_REQUESTED** [review]

File: development/custom-image/install-deps-debian.sh (hunk 1 above)
Comment (barchw) [reply development/custom-image/install-deps-debian.sh]: GO_VERSION moved

File: development/custom-image/install-deps-debian.sh (hunk 2 above)
Comment (barchw) [reply development/custom-image/install-deps-debian.sh]: Exports removed

**Review by Halamix2 — APPROVED** [review]

**Pull request merged**